app/models/claims/claim_activity.rb

```diff
@@ -23,6 +23,8 @@ class Claims::ClaimActivity < ApplicationRecord
   belongs_to :user, class_name: "Claims::SupportUser"
   belongs_to :record, polymorphic: true
 
+  validates :action, presence: true
```

Comment:
Alternatively, you could use the enum validation too.

![CleanShot 2025-01-08 at 16 03 17@2x](https://github.com/user-attachments/assets/547267b7-075e-485b-9183-9a10d37bddd1)